### ISSUE TITLE
Failing to install consul client when installing into a second namespace

### ISSUE THREAD

**Issue by phroggyy** [hashicorp/consul-helm#126]
When installing multiple copies of consul-helm into separate namespaces, the client cannot be installed as it's already installed by another release. Naturally, setting `client.enabled: false` works. Is this the recommended solution?

Note that these secondary namespaces are only for dev in our case – we want each developer to have their own independent setup of the app so they can mess around freely with configs without affecting other developers/staging.

I assume since consul servers are just regular agents that will also respond to requests, it should be fine just running the servers, but I just want to be certain that I won't run into nasty side effects with this approach.

So, in short: if we want to run this chart in multiple namespaces (in the same cluster), is it best to simply turn off the client?

Relates to #110

**Comment by python138** [comment]
Can this chart really support multiple namespaces if it deploys clients as a Daemonset?

**Comment by svlady** [comment]
Actually, you won’t be able to run multiple versions of this chart on the same k8s cluster. This is expected and by design. The reason is that client pods running as a DaemonSet are claiming certain ports on each k8s node. When you deploy a second DaemonSet it will check that those ports are taken and will not deploy new pods. You won't see any issues or warnings - the number of pods in a second DaemonSet will just stay 0 forever.

**Comment by lkysow** [comment]
Hi, yes unfortunately because we use a daemonset and hostPorts it's not possible to install Consul into multiple Kubernetes namespaces when clients are enabled.
